simln-lib/src/sim_node.rs

```diff
@@ -487,38 +487,40 @@ pub trait SimNetwork: Send + Sync {
     fn list_nodes(&self) -> Vec<NodeInfo>;
 }
 
+type LdkNetworkGraph = NetworkGraph<Arc<WrappedLog>>;
```

Comment:
nitty nit: would have moved this refactor into its own commit for the change - single responsibility commits nice where we can get them